lib/service/service.go

```diff
@@ -2065,6 +2067,26 @@ func (process *TeleportProcess) initSSH() error {
 	return nil
 }
 
+func compareVersions(c *Connector) error {
+	resp, err := c.Client.Ping(context.TODO())
```

Comment:
Maybe we should pass a context from the caller here? `reconnectToAuthService` uses `process.ExitContext` for the same call.